### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by bedri**
This pull request cleans up unused files like vim temporary files and adds vim tmp files (*.*~) exception to gitignore.

### CODE REVIEW

File: src/masternodeman.cpp [hunk 1]
```diff
@@ -814,19 +814,6 @@ void CMasternodeMan::ProcessMessage(CNode* pfrom, std::string& strCommand, CData
         CMasternodeBroadcast mnb;
         vRecv >> mnb;
 		
-/*		if (sporkManager.IsSporkActive(SPORK_22_BAN_DUPLICATE_MN_PER_IP)) {
```
Comment (crypt0p3t3r) [inline src/masternodeman.cpp:817]: Why do we have this spork commented out? SPORK_22_BAN_DUPLICATE_MN_PER_IP

**Comment by crypt0p3t3r** [comment]
image

can we remove also these ones plz

File: share/genbuild.sh [hunk 2]
```diff
@@ -37,7 +37,7 @@ if [ "${BITCOIN_GENBUILD_NO_GIT}" != "1" -a -e "$(which git 2>/dev/null)" -a "$(
 
     # otherwise generate suffix from git, i.e. string like "59887e8-dirty"
     SUFFIX=$(git rev-parse --short HEAD)
-    git diff-index --quiet HEAD -- || SUFFIX="$SUFFIX-dirty"
+    git diff-index --quiet HEAD -- || SUFFIX="$SUFFIX-release"
```
Comment (crypt0p3t3r) [inline share/genbuild.sh]: can you revert this please and add the release suffix, I think that this can do it

image

Comment (crypt0p3t3r) [reply share/genbuild.sh]: can you pull master into this branch to sync them together, plz, and solve any conflict that would appear

Comment (bedri) [reply share/genbuild.sh]: Done and committed.

**Comment by crypt0p3t3r** [comment]
pull master into this branch plz

**Comment by bedri** [comment]
I merged master into development.

**Pull request merged**